src/cmlibs/exporter/webgl.py

```diff
@@ -38,6 +40,17 @@ def export(self, output_target=None):
         if output_target is not None:
             self._output_target = output_target
 
+        if self.multiple_levels:
+            self.tessellation_level = "medium"
+            self._document.setTessellationLevel("medium")
+            self.export_webgl()
+
+            self.tessellation_level = "high"
+            self._document.setTessellationLevel("high")
+            self.export_webgl()
+
+        self.tessellation_level = None
+        self._document.setTessellationLevel("low")
```

Comment:
If we are not specifying multiple tessellation levels then we won't be setting the tessellation level to `low`.